url.bs

```diff
@@ -169,9 +169,11 @@ contains bytes that are not <a>ASCII bytes</a> might be insecure and is not reco
 <p>The <dfn oldids=simple-encode-set>C0 control percent-encode set</dfn> are the <a>C0 controls</a>
 and all <a>code points</a> greater than U+007E (~).
 
+<p>The <dfn>fragment percent-encode set</dfn> is the <a>C0 control percent-encode set</a> and
+U+0022 SPACE, U+0022 ("), U+003C(&lt;), U+003E (&gt;), and U+0060 (`).
```

Comment:
U+0020 SPACE.

Reply:
Yup. That was super-dumb. Fixed in https://github.com/whatwg/url/pull/347/commits/d1d85b24579f29555a1b327ad56d85cc77bdfe9a.